packages/serverless-orchestration/test/ServerlessSpoke.js

```diff
@@ -178,8 +178,8 @@ contract("ServerlessSpoke.js", function(accounts) {
     const invalidConfigResponse = await sendRequest(invalidConfigBody);
     assert.equal(invalidConfigResponse.res.statusCode, 500); // error code
     // Expected error text from an invalid path
-    assert.isTrue(invalidConfigResponse.res.text.includes("Bad environment variables! Specify an `EMP_ADDRESS`"));
-    assert.isTrue(lastSpyLogIncludes(spy, "Bad environment variables! Specify an `EMP_ADDRESS`")); // Check the process logger contained the error.
+    assert.isTrue(invalidConfigResponse.res.text.includes("Bad environment variables! Specify an EMP_ADDRESS"));
+    assert.isTrue(lastSpyLogIncludes(spy, "Bad environment variables! Specify an EMP_ADDRESS")); // Check the process logger contained the error.
```

Comment:
do these errors need to be updated?

Reply:
nope.  I think this is correct. I made a small change to the logs that the bots create to accommodate a new (and optional) env variable `FINANCIAL_CONTRACT_ADDRESS` but left the old one as optional to not break the API of the bots configs.

Reply:
+1